src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SDSEarlyReleaseDefaultingRulesService.kt

```diff
@@ -83,8 +83,15 @@ class SDSEarlyReleaseDefaultingRulesService(
     if (dates.containsKey(ReleaseDateType.CRD) || dates.containsKey(ReleaseDateType.ARD)) {
       val controllingDate = getControllingDate(dates)
 
-      // override HDCED to the PRRD if the PRRD is later than the HDCED
+      // Override HDCED or HDCED4PLUS to PRRD if PRRD is later than HDCED or HDCED4PLUS
```

Comment:
not nice - but removing the hdc4 commencement logic would mean we dont need any reference to HDCED4PLUS